docs/future/core/reference/corectl.md

```diff
@@ -228,6 +231,52 @@ Argument:
 
 * **address**: The listen address, in `host:port` format, to be added to the allowed members list.
 
+
+### `get`
+
+Retrieves the current value of a configuration option.
+
+```
+corectl get [key]
+```
+
+Argument:
+
+* **key**: The configuration option to retrieve.
+
+
+### `add`
+
+Adds a tuple to a configuration option's set of values. The configuration option must be defined as a set. The number of value arguments provided must match the configuration option's tuple length. Add will error if a conflicting tuple already exists in the set. To overwrite the existing tuple, provide the `-u` flag.
```

Comment:
Got it. I think the scalar-ness or vector-ness of a given config option is vague with both the original and the suggestion.

How about:

> A configuration option is identified by a key, and consists of either a single tuple or a set of tuples. If a configuration option can have a set of tuples, each tuple in the set should contain the same number of elements.